docs/usage/configuration-options.md

```diff
@@ -2317,6 +2317,11 @@ However there are cases where updates might be desirable - e.g. if you have conf
 This defaults to `true`, meaning that Renovate will perform certain "desirable" updates to _existing_ PRs even when outside of schedule.
 If you wish to disable all updates outside of scheduled hours then configure this field to `false`.
 
+## updatePinnedDependencies
+
+This defaults to `true`.
+Set to false to disable pinned dependencies updates.
```

Comment:
```suggestion
By default, Renovate will attempt to update all detected dependencies, regardless of whether they are defined using pinned single versions (e.g. `1.2.3`) or constraints/ranges (e.g. (`^1.2.3`).
You can set this option to `false` if you wish to disable updating for pinned (single version) dependencies specifically.
```

This is the text that @rarkins wanted to add, but it got "resolved" due to the changes in the markdown file that were from me... 😄

Reply:
I think in this particular case it was resolved due to me resolving it before pushing the actual commit(I click through them as I apply fixes to not forget anything.)

Reply:
(It is rare for me to see such online activity on PRs and instead people usually react half a day later when everything is in order already.)